lift_adapter_template/lift_adapter_template/lift_adapter_template.py

```diff
@@ -67,7 +67,7 @@ def update_callback(self):
         new_state = self._lift_state()
         if new_state is None:
             self.get_logger().error(
-                'Unable to get new state from lift {}'.format(self.lift_guid))
```

Comment:
Should this lift_guid come from the lift state message? Or can it just be the name supplied to it from the adapter?

Reply:
Yup the name is fine, I think I may have glossed over it without checking, thanks for the fix!